lib/modules/category_usage_utils.rb

```diff
@@ -0,0 +1,11 @@
+module CategoryUsageUtils
```

Comment:
Can we remove this file?

Reply:
Leaving as a placeholder for "most popular categories"  query that we will need in the new home page feature branch.